WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsRows.swift

```diff
@@ -154,6 +154,37 @@ enum ProductSettingsRows {
         let cellTypes: [UITableViewCell.Type] = [SettingTitleAndValueTableViewCell.self]
     }
 
+    struct EnableReviews: ProductSettingsRowMediator {
```

Comment:
I'm not sure about using `EnableReviews` because the "Enable" is a verb and this makes it sound like a function. 🤪 Maybe `ReviewsAllowed`? Up to you. 🙂

Reply:
It make sense, updated here 0d522b894444c84289b0a50feb7cc20130509462 😃